simtools/applications/validate_file_using_schema.py

```diff
@@ -94,10 +125,12 @@ def main():
     logger = logging.getLogger()
     logger.setLevel(gen.get_log_level_from_user(args_dict["log_level"]))
 
-    if any(args_dict["file_name"].endswith(ext) for ext in (".yml", ".yaml", ".json")):
+    if args_dict["validate_metadata"]:
+        _validate_metadata(args_dict, logger)
+    elif any(args_dict["file_name"].endswith(ext) for ext in (".yml", ".yaml", ".json")):
```

Comment:
I was actually expecting we would be able to validate both data and metadata in a single shot. But this `elif` means that if the metadata was already validated, the data will not be validated. I suggest allowing both data and metadata to be validated together, even if the metadata validated is the one that comes from the data file (or url). For that, I think that simply replacing the `elif`s by `if`s would do the work.

Reply:
This is one remaining comment. Otherwise all good from my side

Reply:
I think I want to keep it how it is, this is expected behavior (it least in my view): it is an application to validate data or metadata, and I would find it confusing when trying to work on validating data in a file and would get errors on the meta data. Note also that this is an application for developers, used in CI workflows or with interfaces, and not by the typical simtools user.

Reply:
ok, maybe it is indeed simpler to keep it separate. My point was due to the fact that the tool is called `validate file`, but when we pass a data file, the tool only validates the data of that file and not the metadata in there (if present). But let's then keep it like this, I won't object and I think we can merge.